<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GuiItemValues</class>
 <widget class="QDialog" name="GuiItemValues" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSave" >
       <property name="text" >
        <string>Applica</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbCancel" >
       <property name="text" >
        <string>Annulla</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="mp_gbValues" >
     <property name="title" >
      <string>Values</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="0" >
       <widget class="QLabel" name="mp_labelValue3" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Helvetica; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Value 3&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="mp_labelValue2" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Helvetica; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Value 2&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="mp_leValue3" >
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="mp_leValue2" >
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QToolButton" name="mp_tbValue3" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QToolButton" name="mp_tbValue2" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QToolButton" name="mp_tbValue1" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mp_leValue1" >
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mp_labelValue1" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Helvetica; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Value 1&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QToolButton" name="mp_tbValue0" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mp_leValue0" >
        <property name="maxLength" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="mp_labelValue0" >
        <property name="minimumSize" >
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Helvetica; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Value 0&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>mp_leValue0</tabstop>
  <tabstop>mp_tbValue0</tabstop>
  <tabstop>mp_leValue1</tabstop>
  <tabstop>mp_tbValue1</tabstop>
  <tabstop>mp_leValue2</tabstop>
  <tabstop>mp_tbValue2</tabstop>
  <tabstop>mp_leValue3</tabstop>
  <tabstop>mp_tbValue3</tabstop>
  <tabstop>mp_pbOk</tabstop>
  <tabstop>mp_pbSave</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
